client/app/widgets/Notifications/index.jsx

```diff
@@ -11,124 +11,128 @@ export type Props = {
 };
 
 export type State = {
-  notifications: string,
-  alreadyMounted: boolean,
-  open: boolean,
+  notifications?: string,
```

Comment:
I'm curious why these props became optional.

Reply:
These lines were because I did something wrong, and it created an error. I have resolved that error by making them optional I am looking into this.